docs/v2/10_webgl-browser.md

```diff
@@ -39,7 +39,7 @@ int networkId = Web3GL.Network();
 
 ### Block Number {#block-number}
 
-Get the current (i.e. latest) block number.
+Get the current (i.e. latest) block number from a specified node using the `JsonRpcProvider` variable.
```

Comment:
I'd probably just leave the description how it was previously.

Reply:
will update, thx